build/configs/imxrt1020-evk/imxrt1020-evk_download.sh

```diff
@@ -111,6 +112,14 @@ function get_partition_index()
 		app | App | APP) echo "1";;
 		micom | Micom | MICOM) echo "2";;
 		wifi | Wifi | WIFI) echo "4";;
+		userfs | Userfs | USERFS)
```

Comment:
Could you adapt this for anothers? kernel, app, micom, wifi and userfs all are the same.